mapie/regression.py

```diff
@@ -209,18 +232,22 @@ def __init__(
         estimator: Optional[RegressorMixin] = None,
         method: str = "plus",
         cv: Optional[Union[int, str, BaseCrossValidator]] = None,
+        test_size: Optional[Union[int, float]] = None,
         n_jobs: Optional[int] = None,
         agg_function: Optional[str] = "mean",
         verbose: int = 0,
```

Comment:
Move optional arguments at the end

Reply:
All arguments are optional except `method` and `verbose`. Should I move `method` and `verbose` at the beginning?